dependencies.gradle

```diff
@@ -70,7 +70,7 @@ def VERSIONS = [
         'org.latencyutils:LatencyUtils:latest.release',
         'org.mockito:mockito-core:latest.release',
         'org.mockito:mockito-inline:latest.release',
-        'org.mongodb:mongo-java-driver:latest.release',
+        'org.mongodb:mongodb-driver-sync:latest.release',
```

Comment:
This reminded me: I wonder what it would take to support instrumenting other drivers than the sync one. It's not something that needs to be covered in this pull request, but I just thought I'd mention it.

Reply:
https://github.com/micrometer-metrics/micrometer/issues/2498